src/main/java/frc/robot/subsystems/swerve/modules/ModuleUtils.java

```diff
@@ -24,14 +24,18 @@ public int getIndex() {
 
     }
 
-    public static String getModuleLogPath(ModuleName moduleName) {
-        return ModuleConstants.LOG_PATH + moduleName + "/";
-    }
+    public enum ModuleType {
+        TALON_FX("TalonFX");
 
-    public static String getModuleAlertLogPath(ModuleName moduleName) {
-        return ModuleConstants.ALERT_LOG_PATH + moduleName + "/";
-    }
+        private final String moduleType;
+        ModuleType(String moduleType) {
+            this.moduleType = moduleType;
+        }
 
+        public String getModuleType() {
+            return moduleType;
+        }
+    }
```

Comment:
I don't like use type as talon or sim or neo. consider to use module position and swerve name for it

Reply:
If the modules name is the same as swerve name what is the point of having it twice in the log path?

Reply:
there will be in log swerve package. Inside it there will be modules package, velocity package, state package...

Reply:
I get why modules is its own package, I just mean why do "TalonFXSwerve/TalonFXModules/" and not "TalonFXSwerve/Modules/"

Reply:
I dont want TalonFXSwerve/TalonFXModules/, this one is better TalonFXSwerve/Modules/.
but i dont want it to be called after the type system (talon, neo, sim) but Swerve1, Swerve2 (they will have better names if there will be more than 1)